src/actions/ServiceGraphActions.ts

```diff
@@ -0,0 +1,19 @@
+import { createAction } from 'typesafe-actions';
+import { CytoscapeClickEvent } from '../components/CytoscapeGraph/CytoscapeGraph';
+
+export enum ServiceGraphActionKeys {
+  GRAPH_OBJECT_CLICKED = 'GRAPH_OBJECT_CLICKED',
+  GRAPH_READY = 'GRAPH_READY'
+}
+
+// synchronous action creators
+export const ServiceGraphActions = {
+  graphObjectClicked: createAction(ServiceGraphActionKeys.GRAPH_OBJECT_CLICKED, (event: CytoscapeClickEvent) => ({
+    type: ServiceGraphActionKeys.GRAPH_OBJECT_CLICKED,
+    ...event
+  })),
+  graphReady: createAction(ServiceGraphActionKeys.GRAPH_READY, (cy: any) => ({
+    type: ServiceGraphActionKeys.GRAPH_READY,
```

Comment:
Let's not use `type` it's a very overloaded term and it's a typescript keyword for `type` alias